src/py/flwr/simulation/app.py

```diff
@@ -105,10 +123,11 @@ def start_simulation(
     num_clients : Optional[int]
         The total number of clients in this simulation. This must be set if
         `clients_ids` is not set and vice-versa.
-    clients_ids : Optional[List[str]]
+    clients_ids : OptionalUnion[List[int],List[str]]]
         List `client_id`s for each client. This is only required if
         `num_clients` is not set. Setting both `num_clients` and `clients_ids`
         with `len(clients_ids)` not equal to `num_clients` generates an error.
+        If list contains `str` values, they will be converted to `int`.
```

Comment:
This implies a breaking change, right?

Reply:
yes, as discussed. The PR title is updated to `break(...)`